### DISCUSSION TITLE
外部库中的属性绑定并通知前台问题

### DISCUSSION THREAD

**Discussion by Vince0911** [BYJRK/DotNet-Discussions#279] [MVVM]
例如当前我有一个相机厂商提供的SDK，新建了一个Camera类型的对象，它有Name，IPAddr，SN等属性，这些属性都是被封装起来的，默认并没有通知前台的功能（INotifyPropertyChanged），那我想在前台逐一绑定这些属性，并实现后台修改时通知前台的功能，该用何种方式优雅的实现呢？

**Comment by BYJRK** [comment]
你可以看一看我的这期视频：https://www.bilibili.com/video/BV1Vz421f7Do/

里面提到了两种常见方式，一个是写 Dto 类，一个是使用 MVVM 社区工具包中的 `SetProperty` 特殊用法

**Reply by Vince0911** [reply]
thanks！
